File: week-02/day-01/draw_triangle.py
# Write a program that reads a number from the standard input, then draws a
# triangle like this:
#
# *
# **
# ***
# ****
#
# The triangle should have as many lines as the number was

a = int(input("Enter a number: "))
for i in range(a):
    for j in range(i + 1):
        print("*", end="")
    print()

File: week-07/day-1/draw-triangle.js
'use strict';

var lineCount = 4;

// Write a program that draws a
// triangle like this:
//
// *
// **
// ***
// ****
//
// The triangle should have as many lines as lineCount is

for(let i = 0; i < lineCount; i++){
    let asterixes = ''
    for(let j = 0; j < i + 1; j++){
        asterixes += '*';
    }
    console.log(asterixes);
}